data/language/en-GB.txt

```diff
@@ -3676,6 +3676,286 @@ STR_6484    :See-Through vegetation toggle
+STR_6606    :Add phase
+STR_6607    :Before
+STR_6608    :At
+STR_6609    :After
+STR_6610    :Disallow early scenario completion:
+STR_6611    :>
+STR_6612    :<
+STR_6613    :Next deadline
+STR_6614    :Previous deadline
+STR_6615    :Before the end of {STRINGID}, year {INT32}:
+STR_6616    :At the end of {STRINGID}, year {INT32}:
+STR_6617    :After {STRINGID}, year {INT32}:
+STR_6618    :{BLACK}None
+STR_6619    :, each with {STRINGID} of {STRINGID} {STRINGID}
+STR_6620    :, each with {STRINGID} between {STRINGID} and {STRINGID}
+STR_6621    : and {STRINGID} of {STRINGID} {STRINGID}
+STR_6622    : and {STRINGID} between {STRINGID} and {STRINGID}
+STR_6623    :{GREEN}✓
+STR_6624    :{RED}X
+STR_6625    :{YELLOW}!
+STR_6626    :{GREEN}Completed!
+STR_6627    :{RED}Failed
+STR_6628    :Most untidy park award
+STR_6629    :Tidiest park award
+STR_6630    :Award for the park with the best roller coasters
+STR_6631    :Best value park award
+STR_6632    :Most beautiful park award
+STR_6633    :Worst value park award
+STR_6634    :Safest park award
+STR_6635    :Best staff award
+STR_6636    :Best park food award
+STR_6637    :Worst park food award
+STR_6638    :Best park toilets award
+STR_6639    :Most disappointing park award
+STR_6640    :Best water rides award
+STR_6641    :Best custom-designed rides award
+STR_6642    :Most dazzling ride colour schemes award
+STR_6643    :Most confusing park layout award
+STR_6644    :Best gentle ride award
+STR_6645    :Confirm
+STR_6646    :Set Current
+STR_6647    :Select a month
+STR_6648    :Select relation to date
+STR_6649    :Date Relation:
+STR_6650    :{WINDOW_COLOUR_2}Month:
+STR_6651    :{WINDOW_COLOUR_2}Year:
+STR_6652    :Enter description of this phase:
+STR_6653    :Edit phase
+STR_6654    :Dateless
+STR_6655    :Before date
+STR_6656    :At date
+STR_6657    :After date
+STR_6658    :During period
+STR_6659    :At end of period
+STR_6660    :After period
+STR_6661    :No deadline
+STR_6662    :During {INT32} years and {INT32} months
+STR_6663    :After {INT32} years and {INT32} months
+STR_6664    :At the end of {INT32} years and {INT32} months
+STR_6665    :Add Phase
+STR_6666    :Research {STRINGID}
+STR_6667    :Select award:
+STR_6668    :Must be recent:
+STR_6669    :{BLACK}Remove this objective
+STR_6670    :Minimal number of rides:
+STR_6671    :Maximum number of rides:
+STR_6672    :Must be different ride types:
+STR_6673    :Finish existing rides:
+STR_6674    :Minimum track length:
+STR_6675    :Maximum track length:
+STR_6676    :Minimum excitement rating:
+STR_6677    :Maximum excitement rating:
+STR_6678    :Minimum intensity rating:
+STR_6679    :Maximum intensity rating:
+STR_6680    :{COMMA2DP32}
+STR_6681    :Do not have any rides with an extreme intensity
+STR_6682    :Warning weeks:
+STR_6683    :Profit: {CURRENCY}
+STR_6684    :Arithmetic sign:
+STR_6685    :Make this a restriction:
+STR_6686    :Number of guests: {INT32}
+STR_6687    :Parkrating: {INT32}
```

Comment:
Shouldn't it have a space between "park" and "rating"?

Reply:
You're right yeah, I'll fix it.
I see in the tests that it's failing various tests because it can't find Award.h, but I'm really not sure as to why it can't find it. Would you perhaps have an idea?